website/source/docs/platform/k8s/helm.html.md

```diff
@@ -0,0 +1,259 @@
+---
+layout: "docs"
+page_title: "Helm - Kubernetes"
+sidebar_current: "docs-platform-k8s-helm"
+description: |-
+  The Consul Helm chart is the recommended way to install and configure Consul on Kubernetes. In addition to running Consul itself, the Helm chart is the primary method for installing and configuring Consul integrations with Kubernetes such as catalog syncing, Connect injection, and more.
+---
+
+# Helm Chart
+
+The [Consul Helm chart](https://github.com/hashicorp/consul-helm)
+is the recommended way to install and configure Consul on Kubernetes.
+In addition to running Consul itself, the Helm chart is the primary
+method for installing and configuring Consul integrations with
+Kubernetes such as catalog syncing, Connect injection, and more.
+
+This page assumes general knowledge of [Helm](https://helm.sh/) and
+how to use it. Using Helm to install Consul will require that Helm is
+properly installed and configured with your Kubernetes cluster.
+
+-> **Important:** The Helm chart is new and
+may still change significantly over time. Please always run Helm with
+`--dry-run` before any install or upgrade to verify changes.
+
+## Using the Helm Chart
+
+To use the Helm chart, you must download or clone the
+[consul-helm GitHub repository](https://github.com/hashicorp/consul-helm)
+and run Helm against the directory. We plan to transition to using a real
+Helm repository soon. When running Helm, we highly recommend you always
+checkout a specific tagged release of the chart to avoid any
+instabilities from master.
+
+Prior to this, you must have Helm installed and configured both in your
+Kubernetes cluster and locally on your machine. The steps to do this are
+out of the scope of this document, please read the
+[Helm documentation](https://helm.sh/) for more information.
+
+Example chart usage:
+
+```sh
+# Clone the chart repo
+$ git clone https://github.com/hashicorp/consul-helm.git
+$ cd consul-helm
+
+# Checkout a tagged version
+$ git checkout v0.1.0
+
+# Run Helm
+$ helm install --dry-run ./
+```
+
+~> **Warning:** By default, the chart will install _everything_: a
+Consul server cluster, client agents on all nodes, feature components, etc.
+This provides a nice out-of-box experience for new users, but may not be
+appropriate for a production setup. Considering setting the `global.enabled`
+value to `false` and opt-in to the various components.
+
+## Configuration (Values)
+
+The chart is highly customizable using
+[Helm configuration values](https://docs.helm.sh/using_helm/#customizing-the-chart-before-installing).
+Each value has a sane default tuned for an optimal getting started experience
+with Consul. Before going into production, please review the parameters below
+and consider if they're appropriate for your deployment.
+
+* <a name="v-global" href="#v-global">`server`</a> - These values are global values that affect multiple components of the chart.
```

Comment:
Should `server` be `global` here? Maybe not because I see references to `server.enabled` below, but also see `global.enabled` above and the description here sounds like you are describing global config not server - specific ones? The Anchors also include `#v-global`.

Reply:
Whew good catch. Yeah.